pallets/pools/src/lib.rs

```diff
@@ -1190,9 +1180,13 @@ pub mod pallet {
 			let total_assets = epoch_nav.checked_add(&epoch_reserve).unwrap();
 			let mut remaining_assets = total_assets;
 			let pool_is_zero = total_assets == Zero::zero();
+			// we are gonna reverse the order
+			// such that prices are calculated from most senior to junior
+			// there by all the remaining assets are given to the most junior tranche
 			let junior_tranche_id = tranches.len() - 1;
 			tranches
 				.iter_mut()
+				.rev()
```

Comment:
I believe we want to be enumerating before the reverse? We use that as a `tranche_id`, so if the goal is to have stable IDs for more-junior tranches this doesn't solve that

Reply:
This would force `junior_tranche_id` to 0, which is also great because it no longer has to be calculated :tada:

Reply:
yeap